### ISSUE TITLE
VMapManager2: could not load *file*.xxx.vmo

### ISSUE THREAD

**Issue by Smerdokryl** [TrinityCore/TrinityCore#23629]
**Description:**

Worldserver does not load _any_ vmap files, instead producing numerous 'VMapManager2: could not load' errors.

The server has been set up in strict accordance with the TC installation guide, and the files themselves have been extracted with no issues. Moreover, worldserver has no problem accessing dbc and map files in the same data directory.

**Expected behaviour:**

VMap files should load properly.

**Branch(es):**

Master

**TC rev. hash/commit:** 

6455f1c

**Operating system:** Windows Subsystem for Linux (Debian)

**Comment by ReyDonovan** [comment]
hm. im use last client 23.07.2019 enUS not problem

**Comment by Smerdokryl** [comment]
@ReyDonovan Me too, I got the client from the Blizzard Launcher

**Comment by Smerdokryl** [comment]
Is there a way to make worldserver output a more detailed description of why the files couldn't be loaded? A specific error?

**Comment by illfated** [comment]
You could lower the conf value to 2 or 1 in worldserver conf:
```
Logger.mmaps=3,Server
```
Maybe you want to lower one or more of the other values too.
You could for example enable this line too:
```
#Logger.maps=3,Console Server
```
Not quite sure which of the other lines to recommend, but maybe you could see if you get anything interesting by lowering these:
```
Logger.root=5,Console Server
Logger.server=3,Console Server
```

**Comment by Smerdokryl** [comment]
@illfated Those settings don't seem to affect the Vmap Manager, because it still doesn't output anything more than 'could not load'

**Comment by illfated** [comment]
OK, fair enough. I don't know the vmap manager well enough to recommend custom error text solutions, but maybe @jackpoz could suggest some code to produce the output you want for those errors.

**Comment by jackpoz** [comment]
I don't follow master development, sorry

**Comment by illfated** [comment]
Fair enough, I thought maybe @Smerdokryl could add a general code snippet usable both in 3.3.5 and master, but I guess that might not work as expected.
